analytics.js

```diff
@@ -1001,6 +990,7 @@ class Analytics {
     if (serverUrl) {
```

Comment:
This check is redundant. Also, take writeKey and serverUrl along with the options in the `initialize` method.

Reply:
done